nodejs/week1/Circle_calculator.js

```diff
@@ -0,0 +1,19 @@
+class Circle {
+  constructor(radius) {
+    this.radius = radius;
+  }
+
+  getDiameter() {
+    return this.radius * 2;
+  }
+  getCircumference() {
+    return Math.PI * this.radius * 2;
+  }
+  getArea() {
+    return Math.PI * this.radius * this.radius;
```

Comment:
You can also get `number` to the power of `x` as `Math.pow(number, x)` or `number**x`. 👍